app/webpacker/components/RegistrationsV2/Register/RegistrationClosedMessage.jsx

```diff
@@ -0,0 +1,48 @@
+import { DateTime } from 'luxon';
+import React, { useEffect, useState } from 'react';
+import { Message } from 'semantic-ui-react';
+import I18nHTMLTranslate from '../../I18nHTMLTranslate';
+
+export default function RegistrationClosedMessage({
+  registrationStart,
+  onTimerEnd,
+}) {
+  const start = DateTime.fromISO(registrationStart);
+  const [timeLeft, setTimeLeft] = useState(start.diff(DateTime.local(), ['days', 'hours', 'minutes', 'seconds']).toObject());
+
+  useEffect(() => {
+    if (timeLeft.days === 0
+      && timeLeft.hours === 0
+      && timeLeft.minutes === 0
+      && timeLeft.seconds === 0) {
+      onTimerEnd();
+    }
+  }, [onTimerEnd, timeLeft.days, timeLeft.hours, timeLeft.minutes, timeLeft.seconds]);
```

Comment:
Can you just use `timeLeft` as a whole as dependency?

Reply:
I had it auto generated by Rubymine, but it looks like eslint doesn't complain if I just use the whole object (also I googled and it looks like it's only an issue if the reference changes, with I think isn't the case here?)